scripts/dev-cli.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
I'm not sure I understand the reason behind this script. It looks quite complicated and a simple
```bash
mesg-daemon start &
daemon=$!
mesg-cli rest-server --laddr tcp://0.0.0.0:1317 &
lcd=$!
mesg-cli orchestrator start &
orchestrator=$!
wait daemon lcd orchestrator
```
might have done the job no?

Reply:
No it's not. The problem with the `wait` command is it will wait for ALL process to stop.

As I said in the description:
>-  if one is dead: it terminates all other processes and exit with an error code
> - if the script is terminated (by docker for example) then its terminates processes and exit without error